### PR TITLE
[Useless Facts] 🚀 Initial Commit w/ full extension

### PR DESCRIPTION
**Pull request by xmok**
## Description

Hello Raycast team 👋.

This is a simple extension to display a random but useless fact via https://uselessfacts.jsph.pl/.

## Screencast

https://github.com/raycast/extensions/assets/30526133/56a25d5a-a2af-4eae-a091-754e838b59d4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/useless-facts/src/random-useless-fact.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+              <Action.CopyToClipboard
+                title="Copy Useless Fact to Clipboard"
+                content={data.text}
+                icon={Icon.Clipboard}
+              />
+              <Action.CopyToClipboard
+                title="Copy All as JSON to Clipboard"
+                content={data.text}
+                icon={Icon.CopyClipboard}
+              />
```
Comment (pernielsentikaer) [inline extensions/useless-facts/src/random-useless-fact.tsx:40]: Seems like they copies the same data

File: extensions/useless-facts/src/today-s-useless-fact.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              <Action.CopyToClipboard
+                title="Copy Useless Fact to Clipboard"
+                content={data.text}
+                icon={Icon.Clipboard}
+              />
+              <Action.CopyToClipboard
+                title="Copy All as JSON to Clipboard"
+                content={data.text}
+                icon={Icon.CopyClipboard}
+              />
```
Comment (pernielsentikaer) [inline extensions/useless-facts/src/today-s-useless-fact.tsx:40]: Same here

**Comment by xmok** [comment]
Hi.

Thanks for your review. I have updated the JSON code accordingly.

Best.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**